src/metax_api/tests/api/rest/base/views/datasets/write.py

```diff
@@ -440,6 +440,22 @@ class CatalogRecordApiWriteCreateTests(CatalogRecordApiWriteCommon):
     #
     #
 
+    def test_issued_date_is_mandatory(self):
```

Comment:
'mandatory' might be a bit misleading since the field is merely filled by metax if not provided. maybe 'issued_date_is_generated' or similar?

Reply:
You're right, its now changed to 'generated'.